### ISSUE TITLE
"follow target" gives Unknown uri scheme 'vscode-remote'

### ISSUE THREAD

**Issue by davidsgrogan** [cursorless-dev/cursorless#1330]
<img width="435" alt="Screen Shot 2023-03-20 at 11 45 03 AM" src="https://user-images.githubusercontent.com/1801875/226436735-d59a3dfe-f7a9-492e-b996-281fb844bcd2.png">

This is a remote workspace. I'm trying to open that header file. When I try "follow down score" or "follow blue odd" or "follow inside pink crunch" I get an error message in the bottom right: `Unknown uri scheme 'vscode-remote'`

Is "follow" the right cursorless command to open a file? If so, do I need to do something on my end to make it work? Or is this just a bug/limitation in cursorless?

Thanks!

**Comment by pokey** [comment]
Thanks for filing!  Does it work when you ctrl-click it? (or cmd-click on mac)

**Comment by davidsgrogan** [comment]
It does work with cmd-click, yeah.

**Comment by pokey** [comment]
@AndreasArvidsson any ideas on this one? I think you're more familiar with our "follow" code than I am

**Comment by AndreasArvidsson** [comment]
Well yes that is a uri scheme we have not added support for. I would expect `file` in that case.

**Comment by davidsgrogan** [comment]
When I try to open a file without being connected to the remote, I get this error

`Unable to read file 'vscode-remote://ssh-remote+dgrogan/home/dgrogan/c/1/src/third_party/blink/renderer/core/layout/ng/ng_block_node.h'`

So I suppose that's the full uri. Maybe something useful can be parsed out of that?

But I just discovered that "define" instead of "follow" works and will open the file in the editor, so this issue is no longer a big deal for me :) I'll just use "define".

Also, I took a few months off from cursorless (and computers overall), but I'm pretty sure I used to use "follow" with this remote and it what open files just like "define" does now. (That may be unhelpful, and I may be misremembering.)
